server/routes/create/review-sentences/createRecallReviewSentencesController.ts

```diff
@@ -0,0 +1,98 @@
+import { Request, Response } from 'express'
+import { Controller } from '../../controller'
+import CreateRecallUrls from '../createRecallUrls'
+import { PersonJourneyParams } from '../../../@types/journeys'
+import { Page } from '../../../services/auditService'
+import RecallService, { DecoratedCourtCase } from '../../../services/recallService'
+import { datePartsToDate, dateToIsoString, maxOf } from '../../../utils/utils'
+import CalculateReleaseDatesService from '../../../services/calculateReleaseDatesService'
+import { SentenceAndOffence } from '../../../@types/recallTypes'
+import {
+  RecallSentenceCalculation,
+  RecordARecallDecisionResult,
+} from '../../../@types/calculateReleaseDatesApi/calculateReleaseDatesTypes'
+
+export default class CreateRecallReviewSentencesController implements Controller {
+  PAGE_NAME: Page = Page.CREATE_RECALL_REVIEW_SENTENCES_AUTOMATED
+
+  constructor(
+    private readonly recallService: RecallService,
+    private readonly calculateReleaseDatesService: CalculateReleaseDatesService,
+  ) {}
+
+  GET = async (req: Request<PersonJourneyParams>, res: Response): Promise<void> => {
+    const { username } = req.user
+    const { prisoner } = res.locals
+    const { nomsId, journeyId } = req.params
+    const journey = req.session.createRecallJourneys[journeyId]!
+
+    const decision = await this.calculateReleaseDatesService.makeDecisionForRecordARecall(
```

Comment:
Loading the decision again here rather than keeping in session. The page says they can make changes if any sentences or offences are wrong and refresh page.